BUILD

```diff
@@ -33,7 +34,7 @@ package(default_visibility = ["//test/integration:__pkg__"])
 kt_jvm_library(
     name = "studio",
     srcs = glob(["*.kt"]),
-    kotlin_compiler_plugin = "@org_jetbrains_compose_compiler//file",
+    plugins = ["//:org_jetbrains_compose_compiler_plugin"],
```

Comment:
`rules_kotlin` now has its own support for importing compiler plugins. We make use of this now instead of our own `kotlin_compiler_plugin` attribute.